front/src/views/Transfers/CreateShipment/CreateShipmentView.tsx

```diff
@@ -39,6 +40,19 @@ export const ALL_ACCEPTED_TRANSFER_AGREEMENTS_QUERY = gql`
   }
 `;
 
+export const ALL_AVAILABLE_BASES_QUERY = gql`
+  query AllAvailableBases($orgId: ID!) {
+    organisation(id: $orgId) {
+      id
+      name
```

Comment:
Do you even need to fetch id and name of the organisation?

Reply:
I pass it as a prop to `CreateShipment.tsx`